docs/user/webapp/webapp.md

```diff
@@ -1,7 +1,7 @@
 # WebApp
 
-This guide explains how you can use the web app to create transformations. If you are using TOSCAna like descriped in [Getting Started](../getting-started.md) the WebApp application is already started and ready to use.
-
+This guide explains how you can use the web app to create transformations.
+If you are using TOSCAna like described in [Getting Started](../getting-started.md) the WebApp application is already started and ready to use.
```

Comment:
In line 3 you say `web app`. In line 4 you say `WebApp application`, which is funny but repetitive.